rmgpy/data/thermo.py

```diff
@@ -859,6 +856,9 @@ def loadGroups(self, path):
         self.groups['longDistanceInteraction_cyclic']   =   ThermoGroups(label='longDistanceInteraction_cyclic').load(os.path.join(path, 'longDistanceInteraction_cyclic.py'  ), self.local_context, self.global_context)
         self.groups['longDistanceInteraction_noncyclic']   =   ThermoGroups(label='longDistanceInteraction_noncyclic').load(os.path.join(path, 'longDistanceInteraction_noncyclic.py'  ), self.local_context, self.global_context)
 
+        self.recordRingGenericNodes()
+        self.recordPolycylicGenericNodes()
```

Comment:
I know they weren't part of the PR, but I noticed these two functions don't have docstrings, would it be possible for you to add ones?